$OpenBSD: patch-configure,v 1.9 2012/10/17 10:03:22 jasper Exp $

Remove confusing and bogus line since introspection support has been removed.

--- configure.orig	Wed Oct 17 11:55:28 2012
+++ configure	Wed Oct 17 11:55:42 2012
@@ -18788,7 +18788,6 @@ echo "
     Use setgid binaries:   ${enable_setgid}
     Scores user:           ${scores_user}
     Scores & setgid group: ${scores_group}
-    Introspection data:    ${enable_introspection}
 "
 
 if grep "$scores_group:" /etc/group > /dev/null; then
